INSTALL

```diff
@@ -30,12 +30,25 @@
 ################################################################################
 
 Installation Instructions for STACCATO
-$> make
-$> make test (edit the CUDD and DSD library paths)
 
+To change the static/shared library type, use the make parameter:
+BUILD_TYPE=shared or BUILD_TYPE=static (static by default).
+You must specify the path to the CUDD directory with:
+CUDD_PATH=example/path/to/cudd.
+
+Example for building shared library:
+$> make BUILD_TYPE=shared CUDD_PATH=example/path/to/cudd
+
+Example for building static library:
+$> make CUDD_PATH=example/path/to/cudd
+
+You can test static library with:
+$> make test CUDD_PATH=example/path/to/cudd
```

Comment:
What is `CUDD_PATH`? Is it a path to sources or to *.so?

What information about CUDD was used at previous (non-modified) STACCATO?

Reply:
CUDD_PATH it is a path to directory with CUDD sources. 
In non-modified STACCATO they used the same information: path to CUDD headers (cudd/include directory).

Reply:
Ok. It should be concretized here what "CUDD directory" is to avoid misunderstanding.